### PR TITLE
WIP: Adding support for FreeBSD

### PR DESCRIPTION
**Pull request by kadecole**
I have worked on a PR to add support for the FreeBSD operating system. Please review and let me know if you have any suggestions or questions. Thanks.

### CODE REVIEW

**Comment by riemers** [comment]
Seems like a nice addition, feel free to tell me when you think its mature, i'll merge it.

**Comment by kadecole** [comment]
Thanks. I found a couple issues I am trying to work through.

File: handlers/main.yml [hunk 1]
```diff
@@ -3,7 +3,13 @@
 - name: restart_gitlab_runner
   service: name=gitlab-runner state={{ gitlab_runner_restart_state }}
```
Comment (fourstepper) [inline handlers/main.yml:4]: kind of unrelated to your patch:

wouldn't it be possible to join the multiple handlers into one based on the service name defined somewhere else?

**Comment by fourstepper** [comment]
What are you stuck on? Do you perhaps need any help/advice?

File: handlers/main.yml (hunk 1 above)
Comment (kadecole) [reply handlers/main.yml:4]: Yes I think this a good suggestion. I did not know if the author wanted such a modification to the entire project.

**Comment by kadecole** [comment]
Sorry, work has kept me busy and I have been unable to dedicate time to resolve and test fully. I hope to have some time soon.

**Comment by guenhter** [comment]
@kadecole any updates on this?

**Pull request closed without merging**